### PR TITLE
Support configuring `:prefix` and `:receive-endpoint` in remote

### PR DESCRIPTION
**Pull request by zeitstein**

### CODE REVIEW

File: src/repley/shadow/remote.cljs [hunk 1]
```diff
@@ -1,22 +1,31 @@
...
-      value]
-     (fetch (str "http://localhost:" port "/receive")
-            {:method  "POST"
-             :mode    mode
-             :headers {"content-type" "application/edn"}
-             :body    (binding [*print-meta* true]
-                        (pr-str value))}))))
+    ([opts value]
+     (let [{:keys [port mode prefix receive-endpoint]} (merge default-fetch-opts opts)]
+       (when (some? receive-endpoint) ;; nil disables the endpoint
```
Comment (zeitstein) [inline src/repley/shadow/remote.cljs:22]: Note that `:receive-endpoint nil` doesn't actually disable the endpoint on the server.

File: src/repley/shadow/preload.cljs [hunk 2]
```diff
@@ -4,9 +4,9 @@
...
-               (get-in [:http :port]))
-      opts (cond-> {:mode "no-cors"}
-             port (assoc :port port))]
-  (add-tap (partial r/send opts)))
+(def build-opts
+  (-> (goog.object/get js/CLOSURE_DEFINES "repley.shadow.build.opts")
+      (js->clj :keywordize-keys true)
+      r/config->fetch-opts))
+
+(add-tap (partial r/send (merge build-opts {:mode "no-cors"})))
```
Comment (zeitstein) [inline src/repley/shadow/preload.cljs:12]: All relevant config options are now passed to the tap fn.

File: src/repley/shadow/remote.cljs (hunk 1 above)
Comment (tatut) [reply src/repley/shadow/remote.cljs:22]: It should never match and thus won't be called if configured to `nil`

Comment (zeitstein) [reply src/repley/shadow/remote.cljs:22]: I don't think that's correct. The `repley.config/config` will return the default endpoint and then `repley-handler` will match against `localhost:port/receive`.

Comment (zeitstein) [reply src/repley/shadow/remote.cljs:22]: Using a keyword like `:repley/disable-endpoint` might make this easier :)

**Pull request merged**